packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/MaintenanceWindow.tsx

```diff
@@ -198,7 +198,7 @@ export const MaintenanceWindow = (props: Props) => {
           <Typography className={classes.sectionText}>
             {isLegacy ? typographyLegacyDatabase : typographyDatabase}
             {database.cluster_size !== 3
-              ? 'For non-HA plans, expect downtime during this window.'
+              ? ' For non-HA plans, expect downtime during this window.'
```

Comment:
```suggestion
            {isLegacy ? typographyLegacyDatabase : typographyDatabase}{' '}
            {database.cluster_size !== 3
              ? 'For non-HA plans, expect downtime during this window.'
              : null}
```
Nit: how we tend to add spaces to avoid modifying the actual copy.